operating_procedures.md

```diff
@@ -83,23 +45,19 @@ FOCUS Operating Procedures
   * FOCUS Group Chair(s)
   * Support Staff
 
-### 2.3.1 FOCUS Group members (FG member)
+### 2.2.1 FOCUS Group members (FG member)
 
   An FG member is any individual creating content or commenting on an issue or pull request.
 
   FG members MUST read the Project documentation (e.g.: this operational document, contribution guidelines, README, Contributing, and Release Planning documents) before attempting to submit an Issue or Pull Request
 
-  FG members are not allowed to fork a project to build a feature that has been rejected by the FG.
-
-  FG members must only contribute to FGs content that is not protected by patents or is not likely to have essential claims made against it. Contributions MUST only come from Members who have signed the CLA.
-
-### 2.3.2 FOCUS Group Maintainer
+### 2.2.2 FOCUS Group Maintainer
   
-  FG Maintainers are a subset of the FOCUS Group members who have been given write access to one or more repositories within the FOCUS Project's Github Organization. They will advance the day-to-day evolution of the specification and related work products.
+  FG Maintainers are a subset of the FOCUS Group members who have been given write access to one or more FOCUS Working Group repositories within the FOCUS Project's Github organization. They will advance the day-to-day evolution of the specification and related work products.
 
   FG Maintainers are responsible for ensuring that the contents of documents and work products accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. 
 
-  Each FG will designate a FG Maintainer or FG Maintainers for that FG. FG Maintainers work on behalf of the Chair.
+  FG Chair(s) will designate a FG Maintainer or FG Maintainers for the FG. FG Maintainer(s) work on behalf of the FG Chair(s).
```

Comment:
"work on behalf of" > "are overseen by" This is not an employment relationship so we need to avoid that interpretation